config/database.yml

```diff
@@ -1,7 +1,6 @@
 default: &default
-  adapter: postgis
   encoding: utf8
-  url: <%= ENV.fetch("DATABASE_URL") %>
+  url: <%= ENV.fetch("DATABASE_URL").sub("postgres://", "postgis://") %>
   pool: 5
```

Comment:
This probably seems a little weird - it serves two purposes over putting `postgis://` in the `DATABASE_URL` environment variable:

1. The check for PostgreSQL running uses it to connect to the database and has no idea what `postgis://` is.

2. The `DATABASE_URL` is set by the terraform config and we'd have to update that before we could deploy but we'd get into a catch-22 situation where we couldn't update the terraform config without the breaking the existing deployment. This way it happens transparently when the new version is deployed.